### DISCUSSION TITLE
Adding recorded live stream to a playlist, workaround

### DISCUSSION THREAD

**Discussion by emtoonst** [AzuraCast/AzuraCast#7219] [Q&A] [answered]
I would like to automatically replay the latest recorded live dj streams at a certain time. I know this is a feature request and planned (see [here](https://features.azuracast.com/suggestions/197480/adding-of-recorded-live-dj-streams-to-a-playlist)). 
Till this is implemented, what would be a way to automatically create this? I was thinking of:

1. Link the folder in docker: -v /home/recordings/:/home/recordings (I do not know the exact folder)
2. Create a cronjob on the server to copy the latest recording to a folder 
3. Create a playlist and add the folder with the file

Could this be a strategy?

**Comment by silvaalex2021** [comment, marked as answer]
Here at our station, we do the following:

In the system administration, under "Storage Locations," in the "Recordings" tab, I edited the line specifying the path "/var/azuracast/stations/<station_name>/recordings" to something like "/var/azuracast/stations/<station_name>/media/livestreams". This way, we only needed to associate the folder "livestreams/<DJ_NAME>" with a playlist and schedule the time. Now, everything recorded by this DJ account will play at the correct time!

Just a note: you should create a cron job on the host server to clean this folder at least once a day if you want only one transmission to be played!
